src/content/docs/ci-insights.mdx

```diff
@@ -104,13 +114,90 @@ Refer to the dedicated page for your specific framework to configure and verify
   >
     Integrate Go with CI Insights.
   </Docset>
+  <Docset
+    title="Jest"
+    path="/ci-insights/test-frameworks/jest"
+    icon={SiJest}
+  >
+    Integrate Jest with CI Insights.
+  </Docset>
+  <Docset
+    title="JUnit"
+    path="/ci-insights/test-frameworks/junit"
+    icon={SiJunit5}
+  >
+    Integrate JUnit with CI Insights.
```

Comment:
That's a lot of repetition. What about removing it?

Reply:
I'll keep it as is for now. Might be a good idea to remove it though